### PR TITLE
disk_gib, node_count, and instance_type

### PR DESCRIPTION
**Pull request by cbodonnell**
Adds parameters for disk_gib, node_count, and instance_type to the `create-cluster` action. Disk will default to 50 and nodes to 1 to align with the behavior of the CLI.

### CODE REVIEW

File: create-cluster/action.yml [hunk 1]
```diff
@@ -18,6 +18,15 @@ inputs:
   ttl:
     description: 'Cluster TTL (duration, max 48h)'
     required: false
+  disk-gib:
+    description: 'Disk size in GiB'
+    required: false
+  node-count:
+    description: 'Number of nodes to provision'
+    required: false
```
Comment (jdewinne) [inline create-cluster/action.yml:26]: Should this be `nodes`? We recently changed this in the cli to be `nodes`.

File: create-cluster/src/index.ts [hunk 2]
```diff
@@ -12,6 +12,9 @@ async function run() {
     const k8sDistribution = core.getInput('kubernetes-distribution');
     const k8sVersion = core.getInput('kubernetes-version');
     const k8sTTL = core.getInput('ttl');
+    const diskGib: number = +(core.getInput('disk-gib') || 50);
+    const nodeCount: number = +(core.getInput('node-count') || 1);
```
Comment (jdewinne) [inline create-cluster/src/index.ts:16]: Why having `50` and `1`. If nothing is set, the default should be defined by vendor-api

**Review by jdewinne — COMMENTED** [review]
Some questions.

File: create-cluster/src/index.ts (hunk 2 above)
Comment (cbodonnell) [reply create-cluster/src/index.ts:16]: Agreed, will change that here. Should we change this in the CLI so we have a single source for defaults? https://github.com/replicatedhq/replicated/blob/main/cli/cmd/cluster_create.go#L31-L32

**Review by jdewinne — APPROVED** [review]

**Pull request merged**